src/pages/components/Nav.js

```diff
@@ -21,17 +32,44 @@ export default function Nav() {
             accent: '#FFF',
           }}
         >
-          <AppBar.Item href="/what-we-do" active={path === '/what-we-do'}>
+          {/* OLD */}
+          {/* <AppBar.Item href="/what-we-do" active={path === '/what-we-do'}>
             what we do
+          </AppBar.Item> */}
+          {/* <AppBar.Item
+            href="/certificate"
+            active={path.startsWith('/certificate')}
+          >
+            certificates
+          </AppBar.Item> */}
+          {/* OLD END */}
+
+          {/* Inactive app item w/ Link since only Link avoids discarding the context */}
```

Comment:
@dc-andysign can we please stop doing this thing where you litter PR's with old content or comments that should be a PR comment? It basically means this just isn't ready for review and makes reading the change much much harder